test/e2e/scheduling/ubernetes_lite.go

```diff
@@ -125,7 +125,7 @@ func getZoneNameForNode(node v1.Node) (string, error) {
 			return value, nil
 		}
 	}
-	return "", fmt.Errorf("Zone name for node %s not found. No label with key %s",
+	return "", fmt.Errorf("zone name for node %s not found. No label with key %s",
```

Comment:
not quite sure for this suggestion, this actually doesn't looks like a error string, it more looks like a sentence.

Reply:
As IDE marks it as error strings format issue, and the error log is one of the return values of this func, so we fix it like this. Perhaps it's no need to fix this, glad to receive a reply from @alculquicondor @ahg-g.

Reply:
`node %s doesn't have zone label`

Reply:
`node %s doesn't have zone label %s`

Reply:
Thanks for the suggestion, and this have been updated already. Waiting for your review with pleasure.